isis/src/base/apps/topds4/topds4.xml

```diff
@@ -68,7 +68,9 @@
           parameter by removing the last extension and adding .cub
         </description>
       </parameter>
+    </group>
 
+    <group name="Data">

```

Comment:
I moved the DATA parameter into this group because it made sense to me, but I'm not 100% on it.